contracts/ColonyFunding.sol

```diff
@@ -39,7 +39,9 @@ contract ColonyFunding is ColonyStorage, DSMath {
     return 100;
   }
 
-  function setTaskManagerPayout(uint256 _id, address _token, uint256 _amount) public isManager(_id) {
+  function setTaskManagerPayout(uint256 _id, address _token, uint256 _amount) public
```

Comment:
Is it weird that this doesn't go through `executeTaskChange`, but the other payouts do? I understand why (we never need a second signature), but would it be easier for people to use if they were all the same?

Reply:
Yeah, it will make it consistent.